### ISSUE TITLE
Linking

### ISSUE THREAD

**Issue by Casyfill** [mrocklin/blog#47]
Hey, I just noticed the links from your blog's main page to the blogposts are broken.

To reproduce:
1. go to `http://matthewrocklin.com/blog/`
2. click on any link and get `404`

**Comment by stsievert** [comment]
:+1: I've noticed the same issue.

**Comment by mrocklin** [comment]
Yup.  Unfortunately I'm low on personal time.  If anyone has thoughts on
how to resolve this those would be welcome.

**Comment by stsievert** [comment]
Looks like https://github.com/mrocklin/blog/pull/48 will resolve this issue. I've tested it locally, and it works.
